app/AccountancyModule/PaymentModule/presenters/PaymentPresenter.php

```diff
@@ -118,7 +119,12 @@ public function actionDefault(int $id): void
             $this['pairButton']->setGroups([$id]);
         }
 
-        $nextVS = $this->model->getNextVS($group->getId());
+        try {
+            $nextVS = $this->model->getNextVS($group->getId());
+        } catch (InvalidArgumentException $exception) {
```

Comment:
Tady by bylo lepší přidat specifickou výjimku `InvalidVariableSymbol`, která bude dědit od `InvalidArgumentException` a tu následně chytat (a vyhazovat z konstruktoru `VariableSymbol`).

Protože takhle chytáme natolik obecnou výjimku, že to může být nějaká jiná chyba a budeme tak skrývat bugy.